### DISCUSSION TITLE
Auto attack bandits/Auto collect quests

### DISCUSSION THREAD

**Discussion by NoCoM1234** [Sau1707/ModernBot#47] [Ideas]
You should have a new tab next to mix named Quest.Thereyouu can activate to auto collect the rewards from the island quests and open quests.On the tab farm you should the option to select Auto attack bandits(attacks with all troops you currently have on the town).

Keep up the good work ,i just got the script and you have done an amazing job!

**Comment by Sau1707** [comment]
Yes would be a good feature. I'm currently quite buzy this period so it's hard to implement

**Reply by NoCoM1234** [reply]
I have 3 questions. Can you make a guide on how to install the bot?1) I just copied the merged javascript and paste it on tampermonkey.I dont know if this is the right way.2)What does the 'Gui' on the autofarm window does, i have it off and what does the settings on the train tab do? 3)I want to help with the project but on Math university we only do matlab.Do you have any online pdf book/notes to suggest to get started on javascript?

**Reply by Sau1707** [reply]
1) [sau1707.github.io/ModernBot/](https://sau1707.github.io/ModernBot/) there is an install button, just click on it
2) I got banned by using the normal method, the GUI was a test with a different way, it open the window and simulate the user click on the screen instead that in the background...got banned anyway...but they know that it's me making bots so that might be the case
3) Matlab is crap ;). Create an empty script and try to make things happended. On tampermonkey add this 

```js
// ==UserScript==
// @name         MyFirstScript
// @author       <your name>
// @description
// @version      1.0.0
// @match        http://*.grepolis.com/game/*
// @match        https://*.grepolis.com/game/*
// ==/UserScript==

(function() {
    'use strict';

    // Your code here...
    console.log("I'm working!")
})();
```

The open the game, and the browser console "Right click" -> "Inspect" -> "Console" and you should see the "I'm working" message. 
Try to change things and then reload the game to add the updated script, at the beginning it's a try and error. Good Luck!

**Reply by Sau1707** [reply]
2.bis The "Passive" check for all the towns, the "Spell" just check the towns that have the call of the sea or the other spell active to speed up training. The buttons with the % I think are a bit broken now...
